.gitignore

```diff
@@ -1 +1,5 @@
 .*.swp
+
+# LuaCov
+luacov.stats.out
+luacov.report.out
```

Comment:
is there a missing end of line?

Reply:
Yes nice catch! Fixed with: https://github.com/kleisauke/lua-vips/commit/b8917b960ad44f1d325d946caedbd47f44ca94c7